src/main/app/src/app/shared/pipes/mimetypeToExtension.pipe.ts

```diff
@@ -0,0 +1,36 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { FileFormat } from "src/app/informatie-objecten/model/file-format";
+
+@Pipe({
+  name: "mimetypeToExtension",
+  standalone: true,
+})
+export class MimetypeToExtensionPipe implements PipeTransform {
```

Comment:
Like I mentioned in JIRA as well, is it not feasible to add this mapping to the already existing `file-format.ts` file? To me it would seem nice to have all these mappings in one place?

That file already contains a mapping the way around it seems? Can we not re-use this?

```
export enum FileFormat {
  PDF = "application/pdf",
  JPEG = "image/jpeg",
  [..]
  ```

Reply:
moved the constant to the file. i cannot move the pipe functionality to another file because a angular pipe is a seperate thing from file-constants